pimpmycause/profiles/models.py

```diff
@@ -1,26 +1,69 @@
 from __future__ import unicode_literals
 
 from django.db import models
-from django.contrib.auth.models import User
```

Comment:
is this the django default? 

Could this be used also to add email field?, if so, why did you decided to use another.. app?

PS: promise this is last question :)